spec/dummy-app/db/migrate/20200929112246_create_recognisable_sessions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRecognisableSessions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :recognisable_sessions do |t|
+      t.string :recognisable_type
+      t.integer :recognisable_id
+      t.string :sign_in_ip
```

Comment:
Looking at the error that has disappeared from `schema.rb`, I think that Devise stores IP addresses as the special `inet` type if the DB supports it (e.g. Postgres). It'd be good to do the same, wanna see how Devise's trackable creates this migration?